index.js

```diff
@@ -166,6 +167,13 @@ function start() { // eslint-disable-line max-statements
     // Set the X-XSS-Protection HTTP header as a basic protection against XSS
     app.use(helmet.xssFilter());
 
+    // Allow some routes cross-origin resource sharing
```

Comment:
Pls. expand this comment to describe why we would want to allow CORS, what use cases, etc.